cli/azd/pkg/project/service_target_dotnet_containerapp.go

```diff
@@ -377,6 +388,18 @@ func (fns *containerAppTemplateManifestFuncs) Parameter(name string) (string, er
 	return valString, nil
 }
 
+// ParameterWithDefault resolves the name of a parameter defined in the ACA yaml definition.
+// The parameter can be mapped to a system environment variable or be default to a value directly.
+func (fns *containerAppTemplateManifestFuncs) ParameterWithDefault(name string, defaultValue string) (string, error) {
+	envVarMapping := scaffold.EnvFormat(name)
+	// map only to system environment variables. Not adding support for mapping to azd environment by design (b/c
+	// parameters could be secured)
```

Comment:
Let's track this somehow if that makes sense to you. I think either adding the issue here directly, or updating the issue to reference this file works.